vundo.el

```diff
@@ -597,6 +597,7 @@ WINDOW is the window that was/is displaying the vundo buffer."
 
 (define-derived-mode vundo-mode special-mode
   "Vundo" "Mode for displaying the undo tree."
+  :interactive nil
```

Comment:
A short comment explaining why interactive is `nil` would be good.

```
;; This minor mode only applies to buffers created by `vundo',
;; there is no need to show this minor mode to the user.
```

Reply:
Great, I will take yours.